Library/CustomConfig.ts

```diff
@@ -0,0 +1,83 @@
+import { ICustomConfig } from "../Library/ICustomConfig";
+import Logging = require('./Logging');
+
+import fs = require("fs");
+
+const APPLICATION_INSIGHTS_CONFIG_PATH = "APPLICATION_INSIGHTS_CONFIG_PATH";
+
+// Azure Connection String
+export const ENV_connectionString = "APPLICATIONINSIGHTS_CONNECTION_STRING";
```

Comment:
Do you need to export all of these?, are they used somewhere else?

Reply:
Yeah it's used in tests so I had to export it. Do you think I should define it in test and remove export here?

Reply:
Is fine to have tests with the actual value instead of class constants, so you can test the class constant value is correct as well